### DISCUSSION TITLE
Publish a folder

### DISCUSSION THREAD

**Discussion by DelmarHuang** [alangrainger/share-note#36] [Vote for upcoming features]
Now if we want to publish all notes in one folder, we need to publish them one by one, and there is no links between them until we add backlinks or a MOC.

Is it possible to publish a folder which will be the home page? Thank you for your hard work!!

**Comment by alangrainger** [comment]
The intent of Share Note is to quickly share individual notes. For folders or for an entire vault you're more likely looking for a plugin which lets you share a "digital garden" similar to the official Publish.

One option would be https://github.com/oleeskild/obsidian-digital-garden

**Reply by DelmarHuang** [reply]
Yes, you have a point! Thank you
